spec/support/suspenders.rb

```diff
@@ -40,23 +85,23 @@ def suspenders_help_command
   end
 
   def setup_app_dependencies
-    if File.exist?(project_path)
-      Dir.chdir(project_path) do
-        Bundler.with_clean_env do
-          `bundle check || bundle install`
-        end
+    Dir.chdir(project_path) do
+      Bundler.with_clean_env do
```

Comment:
This pattern of `Dir.chdir(project_path) { Bundler.with_clean_env { … } }` seems to be repeated quite a lot. How about a `#run_in_project` method?